lib/common/session_model.dart

```diff
@@ -183,9 +178,21 @@ class SessionModel extends Model {
     if (isMobile()) {
       return subscribedSingleValueBuilder<String>('lang', builder: builder);
     }
+    final websocket = WebsocketImpl.instance();
     return ffiValueBuilder<String>(
       'lang',
       defaultValue: 'en',
+      onChanges: (setValue) {
+        if (websocket == null) return;
+        websocket.messageStream.listen(
+          (json) {
+            if (json["type"] != "pro") return;
```

Comment:
Not super important, but Do you think we can create a utils method for  `if (json["type"] != "pro") return;` check we are starting to use this at multiple places, In the future if the needed change we need to change only one?

Reply:
Done!